packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -109,6 +110,67 @@ export default class ComponentsTableRoute extends Route {
   />
   {{! template-lint-enable no-unbalanced-curlies }}
   {{! prettier-ignore-end }}
+  <h4 class="dummy-h4">Sortable Table</h4>
+  <p class="dummy-paragraph">For the sortable table, the invocation and use is a little bit different:</p>
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">1. Shape the data model for use; in this example we've placed it in the page's route.</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="javascript"
+    @code="// app/routes/components/table.js
+
+import Route from '@ember/routing/route';
+import { capitalize } from '@ember/string';
+
+export default class ComponentsTableRoute extends Route {
+  async model() {
+    let response = await fetch('/api/folk.json');
+    let { data } = await response.json();
+
+    // make sure the variable is declared outside of the loop
+    // so we can return it in the model response
+    let columns;
+    let dataResponse = data.map((model) => {
+      let { id, attributes } = model;
+      columns = Object.keys(attributes);
+      return { id, ...attributes };
+    });
+    columns = columns.map((column) => {
+      return { key: column, label: capitalize(column) };
+    });
+    return { data: dataResponse, columns };
+  }
+}"
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">2. Invoke the
+    <span class="dummy-code">Hds::Table::TableSortable</span>
+    component in your template file.</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="handlebars"
```

Comment:
Question: For this kind of example is it best to specify "handlebars" for @language vs. "markup"? For the Tabs component I'm working on, I specified @language as "markup" following what I saw in another component's documentation.

Reply:
I'll double check on this 👍

Reply:
Thanks!